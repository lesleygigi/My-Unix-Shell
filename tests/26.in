grep file2 tests/test_dir/file2 | tee /tmp/grep_result | sort | uniq | wc -l
cat /tmp/grep_result
