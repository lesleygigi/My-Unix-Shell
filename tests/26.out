grep file2 tests/test_dir/file2 | tee /tmp/grep_result | sort | uniq | wc -l
3
cat /tmp/grep_result
content of file2
content of file2
content of file2
file2: more lines in file2
the last line of file2
